Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:14
+FROM node:16-alpine
```

Comment:
Is the docker image used any more? Local development maybe?

Reply:
`16-alpine` is [still listed as supported](https://hub.docker.com/_/node), so I assume it's still working? Let me run this on Docker local to see if there are any hiccups, but it should be no worries.

Reply:
I've changed it to `node:16`.